src/main/java/aquality/selenium/core/utilities/ISettingsFile.java

```diff
@@ -0,0 +1,49 @@
+package aquality.selenium.core.utilities;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Describes reader of settings file.
+ */
+public interface ISettingsFile {
+
+    /**
+     * Gets value from settings file.
+     *
+     * @param path Path to the value. Depending on file format, it can be jsonPath, xpath etc.
+     * @return Value.
+     */
+    Object getValue(String path);
```

Comment:
can we have generic return type here?

Reply:
I think yes, but I am not sure it would be easy to update.
Let me create new issue and we will update it later. 
For now, I just copied the class to this project

Reply:
#19